cdap-api/src/main/java/co/cask/cdap/api/dataset/lib/PartitionedFileSetArguments.java

```diff
@@ -170,4 +172,15 @@ public static Comparable convertFieldValue(String where, String kind, String fie
                       where, kind, stringValue, fieldName, fieldType.name()), e);
     }
   }
+
+  public static void addPartitions(Map<String, String> arguments, Iterator<Partition> partitionIterator) {
```

Comment:
Or is this meant to be _in addition_ to the method that adds an input partition key?